releases/7.6.10.md

```diff
@@ -28,7 +29,24 @@ JavaScript:
     pip install --upgrade arches==7.6.10
     ```
 
-3. If you are running Arches on Apache, restart your server:
+3.  If you are upgrading from version 7.6.9, you'll need to rebuild your frontend bundle:
```

Comment:
Is this strictly true? The frontend changes are addressing edge cases that caused bundles to not build at all, so if you've got a bundle, wouldn't you be all set?

Reply:
Ah, I take it the arches apps were omitted, perhaps silently? Then let's leave this in. 👍